libr/core/cmd_debug.inc.c

```diff
@@ -1393,6 +1393,48 @@ static void cmd_debug_modules(RCore *core, int mode) { // "dmm"
 	r_list_free (list);
 }
 
+R_API double get_glibc_version(RCore *core, const char *libc_path) {
+	double version = 0.0f;
+	RBin *bin = r_bin_new ();
+	RIO *io = r_io_new ();
+	r_io_bind (io, &bin->iob);
+	RBinFileOptions opt = { 0 };
+
+	if (r_bin_open (bin, libc_path, &opt)) {
+		RBinString *string;
+		RList *strings = r_bin_get_strings (bin);
+		RListIter *iterator;
+
+		// I feel like this regex might miss some cases
+		// but it seems to be the same one used by GEF/pwndbg
+		RRegex *glibc_version_regex = r_regex_new ("[0-9]+[0-9]*\\.[0-9]+[0-9]*", "e");
+		r_list_foreach (strings, iterator, string) {
+
+			/*
+			  This logic is to search for the following two strings in glibc
+			  and parse out the version:
+			  1) "glibc <version>"
+			  2) "GNU C Library (GNU libc) stable release version <version>."
+			*/
+			if ((strstr (string->string, "glibc") || strstr (string->string, "GNU C Library"))) {
+				RList *matches = r_regex_match_list (glibc_version_regex, string->string);
+
+				// We only care about the first match
+				char *first_match = r_list_first (matches);
+				version = r_num_get_float (core->num, first_match);
```

Comment:
i think this api should be renamed to get_double instead 🤔 

```unum.c:R_API double r_num_get_float(RNum *num, const char *str) {```

Reply:
I can make this change, but I'd rather do it in a separate PR

Reply:
Yes please